telemetry-firmware/Core/Src/user.cpp

```diff
@@ -46,6 +48,7 @@ static constexpr uint32_t speed_control_period = 10;
 
 void CPP_UserSetup(void)
 {
+	SEGGER_RTT_ConfigUpBuffer(0, NULL, NULL, 0, SEGGER_RTT_MODE_);
```

Comment:
Build is not passing

```suggestion
	SEGGER_RTT_ConfigUpBuffer(0, NULL, NULL, 0, SEGGER_RTT_MODE_H);
```


> ../Core/Src/user.cpp:51:46: error: 'SEGGER_RTT_MODE_' was not declared in this scope; did you mean > 'SEGGER_RTT_CONF_H'?
>   51 |  SEGGER_RTT_ConfigUpBuffer(0, NULL, NULL, 0, SEGGER_RTT_MODE_);
>      |                                              ^~~~~~~~~~~~~~~~
>      |                                              SEGGER_RTT_CONF_H
> make: *** [Core/Src/subdir.mk:54: Core/Src/user.o] Error 1